website/docs/r/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -73,6 +73,9 @@ The following arguments are supported:
 - `resource_group` - (Optional, string,ForceNew)The resource group id
 - `tags` - (Optional, array of strings) Tags associated with the instance.
 
+**NOTE**: For `ips` , do not provide `id` and `subnet` together. Provider either one of them per configuration block.
+**NOTE**: `ips` once configured inline in this resource are not modifiable.  Prefer using `ibm_is_virtual_endpoint_gateway_ip` resource to bind/unbind new reserved IPs to endpointgateways and use the resource `ibm_is_subnet_reserved_ip` to create new reserved IPs
```

Comment:
**NOTE**: `id` and `subnet` are mutually exclusive. 
And please move this under `ips` block

Reply:
```
**NOTE**: `ips` configured inline in this resource are not modifiable. Prefer using `ibm_is_virtual_endpoint_gateway_ip` resource to bind/unbind new reserved IPs to endpoint gateways and use the resource `ibm_is_subnet_reserved_ip` to create new reserved IP. 
```
Made some changes to the block